src/main/kotlin/io/github/codeocean_scraper/Main.kt

```diff
@@ -6,31 +6,31 @@ import java.net.URL
 import java.nio.file.Paths
 
 
-fun printUsage() {
-    println("Usage: codeocean_scraper targetDirectory statisticsUrl")
+private fun printUsage() {
+    println("Usage: codeocean_scraper baseUrl email password exerciseId targetDirectory")
     System.exit(1)
 }
 
+private fun URL.ensureHttps() = when (protocol) {
+    "https" -> this
+    "http" -> URL(toString().replaceFirst("http", "https"))
```

Comment:
There is a constructor for `URL` but it lacks a parameter for the `#reference` part of the url, so this seemed like the easiest way to preserve everything.